### PR TITLE
California TANF resources variable inputs

### LINKED ISSUE #3791: Adjust the `ca_tanf_resources` variable inputs
In #3788 we are removing the list from the file which should be re-added with the correct inputs based on the reference

### PR DESCRIPTION
**Pull request by vrathi101**
Fixes #3791

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+description: California CalWORKs counts these property sources as resources.
+
+metadata:
+  unit: list
+  period: year
+  label: California CalWORKs property breakdown
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml:6]:
```suggestion
  label: California CalWORKs TANF resoruces
```

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  reference:
+    - title: Los Angeles Department of Public Social Services E-Policy | Property Definitions
+      href: http://epolicy.dpss.lacounty.gov/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/index.htm?&area=general&type=responsivehelp&ctxid=&project=ePolicyMaster#t=mergedProjects%2FCalWORKs%2FCalWORKs%2F42-200_Property%2F42-200_Property.htm%23Definitionsbc-4&rhtocid=_3_1_2_0_3
+values:
+  2024-01-01:
+    - real_property
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml:12]: I think this would be grouped as property value

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml (hunk 2 above)
Comment (vrathi101) [reply policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml:12]: The reference distinguishes between each type of property (as these are the resource constituents). Should we do the same or combine into one?

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml:12]: We should combine them

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  unit: list
+  period: year
+  label: California CalWORKs TANF resources
+  reference:
+    - title: Los Angeles Department of Public Social Services E-Policy | Property Definitions
+      href: http://epolicy.dpss.lacounty.gov/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/index.htm?&area=general&type=responsivehelp&ctxid=&project=ePolicyMaster#t=mergedProjects%2FCalWORKs%2FCalWORKs%2F42-200_Property%2F42-200_Property.htm%23Definitionsbc-4&rhtocid=_3_1_2_0_3
+values:
+  2024-01-01:
+    - property_value
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml:12]: image

I missed this:
The personal property includes other items such as cash etc. we need to see how much of that we can include given inputs that we have

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  unit: list
+  period: year
+  label: California CalWORKs TANF resources
+  reference:
+    - title: Los Angeles Department of Public Social Services E-Policy | Property Definitions
+      href: http://epolicy.dpss.lacounty.gopv/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/index.htm?&area=general&type=responsivehelp&ctxid=&project=ePolicyMaster#t=mergedProjects%2FCalWORKs%2FCalWORKs%2F42-200_Property%2F42-200_Property.htm%23Definitionsbc-4&rhtocid=_3_1_2_0_3
+values:
+  2024-01-01:
+    - property_value
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml:12]:
```suggestion
    - assessed_property_value
```

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+metadata:
+  unit: list
+  period: year
+  label: California CalWORKs TANF resources
+  reference:
+    - title: Los Angeles Department of Public Social Services E-Policy | Property Definitions
+      href: http://epolicy.dpss.lacounty.gopv/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/index.htm?&area=general&type=responsivehelp&ctxid=&project=ePolicyMaster#t=mergedProjects%2FCalWORKs%2FCalWORKs%2F42-200_Property%2F42-200_Property.htm%23Definitionsbc-4&rhtocid=_3_1_2_0_3
+values:
+  2024-01-01:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml:11]:
```suggestion
  2022-07-01:
```

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+description: California CalWORKs counts these property sources as resources.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml:1]:
```suggestion
description: California counts the following resources under the CalWORKs program.
```

**Review by PavelMakarchuk — APPROVED** [review]

**Comment by PavelMakarchuk** [comment]
@MaxGhenis would be good to pass on to SBN once merged

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**